src/pynwb/io/utils.py

```diff
@@ -0,0 +1,27 @@
+import re
+from typing import Tuple
+
+from hdmf.build import Builder
+
+
+def get_nwb_version(builder: Builder) -> Tuple[int, ...]:
+    """Get the version of the NWB file from the root of the given builder, as a tuple.
+
+    If the "nwb_version" attribute on the root builder equals "2.5.1", then (2, 5, 1) is returned.
+    If the "nwb_version" attribute on the root builder equals "2.5.1-alpha", then (2, 5, 1) is returned.
```

Comment:
Why is the ``alpha`` being dropped in the version? It seems strange that "2.5.1" and "2.5.1-alpha" both translate to the same version?

Reply:
The version tuple is used to branch the API behavior based on the version, e.g., do X if version < 2.5.1, else Y. Version "2.5.1-alpha" should appear only in development. The features in schema "2.5.1-alpha" will be included in "2.5.1" and so for this purpose, the downstream behavior should treat "2.5.1-alpha" and "2.5.1" as equivalent. 

We could refactor this function so that a suffix like "-alpha" is also returned. In Python `(2, 5, 1, "-alpha") > (2, 5, 1)`, so the branching code above should still work.

Reply:
I would suggest to maybe add a flag (e.g., ``include_prerelease=False``) so that a user can configure whether the prerelease tag (alpha, beta etc.) should be included or not.